crates/warpgate/src/loader.rs

```diff
@@ -76,6 +80,7 @@ impl PluginLoader {
         &self,
         id: I,
         locator: L,
+        registries: Vec<OciRegistry>,
```

Comment:
Instead of passing the registries to this function, add `add_registry` and `add_registries` methods to `PluginLoader`.